meg_runtime/locking/manager.py

```diff
@@ -0,0 +1,100 @@
+"""MEG system file locking
+
+To be used to lock files, unlock files, override locks, and view locks
+Will confirm user roles and preform required git operations
+
+All file paths are relitive to the repository directory
+Working directory should be changed by the git module
+"""
+
+import os
+from meg_runtime.locking.lockFile import LockFile
+from meg_runtime.logger import Logger
+
+LOCKFILE_DIR = ".meg" + os.sep
+LOCKFILE_NAME = "locks.json"
```

Comment:
Move these inside the class instance so they don't pollute the global namespace.